lib/src/sv_tlv.c

```diff
@@ -876,7 +870,6 @@ encode_signature(signed_video_t *self, uint8_t *data)
   // Write info field
   write_byte(last_two_bytes, &data_ptr, gop_info->encoding_status, epb);
   // Write hash type
-  write_byte(last_two_bytes, &data_ptr, (uint8_t)gop_info->signature_hash_type, epb);
```

Comment:
Simply removing data from the TLV will break backward compatibility. Use the `version` to handle both cases.